schedule.md

```diff
@@ -3,64 +3,138 @@ layout: page
+* [Introduction](http://software-carpentry.org/v4/databases/intro.html)
 
-[Assignment 1]({{ site.baseurl }}/assignments/programming-1)
+### Week 2: Selecting, Filtering, Sorting, and Aggregating Data
 
-### Week 2: Strings, Methods
+#### Material:
+* [Selecting](http://software-carpentry.org/v4/access/select.html)
+* [Filtering](http://software-carpentry.org/v4/access/filter.html)
+* [Sorting](http://software-carpentry.org/v4/access/sort.html)
+* [Aggregation](http://software-carpentry.org/v4/access/aggregation.html)
+* [Nulls](http://software-carpentry.org/v4/access/null.html)
 
-#### Reading:
+[Assignment 1](/assignments/sql-1)
 
-* [Strings](http://nbviewer.ipython.org/urls/github.com/ethanwhite/progbio/raw/master/ipynbs/strings.ipynb)
+### Week 3: Tidy Data, Joins, Nested Queries
 
-[Assignment 2]({{ site.baseurl }}/assignments/programming-2)
+#### Material:
 
-### Week 3: Functions, for loops, lists
+* [Tidy Data, Sections 1-3](http://vita.had.co.nz/papers/tidy-data.pdf)
+* [Joins](http://software-carpentry.org/v4/access/join.html)
+* [Nested Queries](http://software-carpentry.org/v4/access/nested.html)
 
-#### Reading:
+[Assignment 2](/assignments/sql-2)
 
-* [SWC Novice Python: Creating Functions](http://software-carpentry.org/v5/novice/python/02-func.html) (M)
-* [SWC Novice Python: Analyzing Multiple Data Sets](http://software-carpentry.org/v5/novice/python/03-loop.html) (W)
+### Week 4: Introduction to R and RStudio
 
-[Assignment 3]({{ site.baseurl }}/assignments/programming-3)
+#### Topics
 
-### Week 4: Making choices, Problem decomposition
+* R & RStudio
+* Expressions & Variables
+* Types
+* Errors
+* Vectors
 
 #### Reading:
 
-* [SWC Novice Python: Making Choices](http://software-carpentry.org/v5/novice/python/04-cond.html)
+* [Getting Started](http://datacarpentry.github.io/R-ecology/00-before-we-start.html)
+* [Introduction to R](http://datacarpentry.github.io/R-ecology/01-intro-to-R.html)
 
-Additional Material: 
-[Problem decomposition](http://programmingforbiologists.org/material/problem-decomposition)
+[Assignment 3](/assignments/r-1)
 
-[Assignment 4]({{ site.baseurl }}/assignments/programming-4)
+### Week 5: Working with Data
 
-### Week 5: Working with tabular data, Debugging, Style
+#### Topics
 
-#### Reading:
+* Matrices & data frames
+* Importing data
+* Basic subsetting
+* Readable code
+  * Functions
 
-* [Pandas - working with data they way your brain wants](http://nbviewer.ipython.org/urls/github.com/ethanwhite/progbio/raw/master/ipynbs/pandas.ipynb)
-  
-[Assignment 5]({{ site.baseurl }}/assignments/programming-5)
+#### Reading
 
-### Week 6: Graphing, Style, Program Design
+* [Starting with data](http://datacarpentry.github.io/R-ecology/02-starting-with-data.html)
+* [Introduction to data frames](http://datacarpentry.github.io/R-ecology/03-data-frames.html)
+* [Creating functions](http://swcarpentry.github.io/r-novice-inflammation/02-func-R.html)
```

Comment:
I wonder about including 'Creating Functions' here. The exercises up through 'Graphing' & 'Statistics' all use built-in functions. So, I thought it would make sense to push off writing functions until after the 'working with data' block. On the other hand, I'm not sure if there is a good resource that separates using functions from writing functions. I think the functions exercises are independent of other material, so they should fit fine with where ever you decide to put them.